components/bounty/page-actions/view.tsx

```diff
@@ -97,7 +95,7 @@ export default function PageActionsView({
                   isCreatePr={isCreatePr}
                   isCreateProposal={isCreateProposal}
                   isExecuting={isExecuting}
-                  handleShowPRModal={handleShowPRModal}
+                  handleShowPRModal={handlePullrequest}
```

Comment:
you removed the modal, would this still make sense, since we don't have the prmodal anymore?